gift-certificate-application/service/src/test/java/com/epam/esm/service/impl/OrderServiceImplTest.java

```diff
@@ -0,0 +1,177 @@
+    private static final long USER_ID = 1L;
+    private static final String FIST_NAME = "Ivan";
+    private static final String LAST_NAME = "Ivanov";
+    private static final String LOGIN = "I-Ivan";
+    private static final String NAME = "Gift certificate";
+    private static final String DESCRIPTION = "description";
+    private static final BigDecimal PRICE = BigDecimal.valueOf(99.99);
+    private static final short DURATION = 100;
+    private static final Long GIFT_CERTIFICATE_ID = 1L;
+    private static final Long TAG_ID = 1L;
+    private static final String TAG_NAME = "Tag";
+    private static final int PAGE = 1;
+    private static final int SIZE = 10;
+    private static final Long ENTITIES_NUMBER = 1L;
+    private OrderDao orderDao;
+    private OrderService orderService;
+    private Order order;
+    private User user;
+    private GiftCertificate giftCertificate;
+    private OrderDto orderDto;
+    private PaginationValidator paginationValidator;
+    private UserService userService;
+    private GiftCertificateService giftCertificateService;
+    private List<Order> orderList;
+    private CustomPagination pagination;
+
+    @BeforeAll
+    void beforeAll() {
+        orderDao = mock(OrderDaoImpl.class);
+        paginationValidator = mock(PaginationValidatorImpl.class);
+        userService = mock(UserServiceImpl.class);
+        giftCertificateService = mock(GiftCertificateServiceImpl.class);
+        paginationValidator = mock(PaginationValidatorImpl.class);
+
+        user = new User();
+        user.setId(USER_ID);
+        user.setFirstName(FIST_NAME);
+        user.setLastName(LAST_NAME);
+        user.setLogin(LOGIN);
+
+        Tag tag = new Tag();
+        tag.setId(TAG_ID);
+        tag.setName(TAG_NAME);
+        List<Tag> tagList = new ArrayList<>();
+        tagList.add(tag);
+
+        giftCertificate = new GiftCertificate();
+        giftCertificate.setId(GIFT_CERTIFICATE_ID);
+        giftCertificate.setName(NAME);
+        giftCertificate.setDescription(DESCRIPTION);
+        giftCertificate.setPrice(PRICE);
+        giftCertificate.setDuration(DURATION);
+        giftCertificate.setTagList(tagList);
+
+        order = new Order();
+        order.setId(ID);
+        order.setUser(user);
+        order.setGiftCertificate(giftCertificate);
+        order.setPrice(giftCertificate.getPrice());
+        order.setCreateDate(LocalDateTime.now());
+
+        orderDto = new OrderDto();
+        orderDto.setUserId(USER_ID);
+        orderDto.setGiftCertificateId(GIFT_CERTIFICATE_ID);
+
+        orderList = new ArrayList<>();
+        orderList.add(order);
+
+        pagination = new CustomPagination();
+        pagination.setPage(PAGE);
+        pagination.setSize(SIZE);
+
+        orderService = new OrderServiceImpl(orderDao, userService, giftCertificateService, paginationValidator);
+    }
+
+    @Test
+    void testFindAll() {
+        List<Order> expectedResult = orderList;
```

Comment:
I suppose you can directly use orderList and do not create an additional variable